seldon-core-operator/rockcraft.yaml

```diff
@@ -0,0 +1,115 @@
+      cd $CRAFT_PART_INSTALL/opt/seldon-core-operator/licenses/mpl_source
+
+      # Per https://github.com/SeldonIO/seldon-core/blob/5591c42b6a40a44641b848d86f9228f623c64598/operator/Dockerfile
+      wget -qO armon-consul-api.tar.gz https://github.com/armon/consul-api/archive/master.tar.gz
+      wget -qO go-sql-driver-mysql.tar.gz https://github.com/go-sql-driver/mysql/archive/master.tar.gz
+      wget -qO hashicorp-consul.tar.gz https://github.com/hashicorp/consul/archive/main.tar.gz
+      wget -qO hashicorp-errwrap.tar.gz https://github.com/hashicorp/errwrap/archive/master.tar.gz
+      wget -qO hashicorp-go-cleanhttp.tar.gz https://github.com/hashicorp/go-cleanhttp/archive/master.tar.gz
+      wget -qO hashicorp-go-immutable-radix.tar.gz https://github.com/hashicorp/go-immutable-radix/archive/master.tar.gz
+      wget -qO hashicorp-go-multierror.tar.gz https://github.com/hashicorp/go-multierror/archive/master.tar.gz
+      wget -qO hashicorp-go-plugin.tar.gz https://github.com/hashicorp/go-plugin/archive/master.tar.gz
+      wget -qO hashicorp-go-retryablehttp.tar.gz https://github.com/hashicorp/go-retryablehttp/archive/master.tar.gz
+      wget -qO hashicorp-go-rootcerts.tar.gz https://github.com/hashicorp/go-rootcerts/archive/master.tar.gz
+      wget -qO hashicorp-go-sockaddr.tar.gz https://github.com/hashicorp/go-sockaddr/archive/master.tar.gz
+      wget -qO hashicorp-go-uuid.tar.gz https://github.com/hashicorp/go-uuid/archive/master.tar.gz
+      wget -qO hashicorp-go-version.tar.gz https://github.com/hashicorp/go-version/archive/main.tar.gz
+      wget -qO hashicorp-golang-lru.tar.gz https://github.com/hashicorp/golang-lru/archive/master.tar.gz
+      wget -qO hashicorp-hcl.tar.gz https://github.com/hashicorp/hcl/archive/main.tar.gz
+      wget -qO hashicorp-logutils.tar.gz https://github.com/hashicorp/logutils/archive/master.tar.gz
+      wget -qO hashicorp-memberlist.tar.gz https://github.com/hashicorp/memberlist/archive/master.tar.gz
+      wget -qO hashicorp-serf.tar.gz https://github.com/hashicorp/serf/archive/master.tar.gz
+      wget -qO hashicorp-vault.tar.gz https://github.com/hashicorp/vault/archive/master.tar.gz
+      wget -qO hashicorp-yamux.tar.gz https://github.com/hashicorp/yamux/archive/master.tar.gz
+      wget -qO mitchellh-cli.tar.gz https://github.com/mitchellh/cli/archive/master.tar.gz
+      wget -qO mitchellh-gox.tar.gz https://github.com/mitchellh/gox/archive/master.tar.gz
+      wget -qO mozilla-tls-observatory.tar.gz https://github.com/mozilla/tls-observatory/archive/master.tar.gz
+
+  non-root-user:
+    plugin: nil
+    after: [seldon-core-operator, mpl-deps]
+    overlay-script: |
+      # Create a user in the $CRAFT_OVERLAY chroot
+      groupadd -R $CRAFT_OVERLAY -g 1001 ubuntu
+      useradd -R $CRAFT_OVERLAY -M -r -u 1001 -g ubuntu ubuntu
+    override-prime: |
+      craftctl default
+      chown -R 1001:1001 opt/seldon-core-operator
```

Comment:
Are these two lines with `chown` necessary ? we are not doing it in other rocks

Reply:
May be we should, actually do it in other rocks as well.
Those are created as owned by root. Changing ownership plays into running as non-root story.